ffi/dotnet/Devolutions.Picky/Generated/RawSshPrivateKey.cs

```diff
@@ -57,7 +57,7 @@ public partial struct SshPrivateKey
     public static unsafe extern IntPtr FromPem(Pem* pem, byte* passphrase, nuint passphraseSz);
 
     [DllImport(NativeLib, CallingConvention = CallingConvention.Cdecl, EntryPoint = "SshPrivateKey_from_private_key", ExactSpelling = true)]
-    public static unsafe extern SshPrivateKey* FromPrivateKey(PrivateKey* key);
+    public static unsafe extern IntPtr FromPrivateKey(PrivateKey* key);
```

Comment:
Do you know what caused the generator to now return an `IntPtr` instead of `SshPrivateKey*`?

Reply:
Yes, it seems like generator uses `IntPtr` instead of `Type*` when the method returns DiplomatResult<...>. It also generates exception handling code for it

Reply:
Oh right. I remember that now :smile: